golang/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/uuid"
+	"github.com/momentohq/client-sdk-go/momento"
+)
+
+func main() {
+	var AuthToken = os.Getenv("MOMENTO_AUTH_TOKEN")
+	const (
+		CacheName             = "momentocache"
+		ItemDefaultTtlSeconds = 60
+	)
+
+	// Initializes Momento
+	client, err := momento.SimpleCacheClient(&momento.SimpleCacheClientRequest{
+		AuthToken:         AuthToken,
+		DefaultTtlSeconds: ItemDefaultTtlSeconds,
+	})
+	if err != nil {
+		fmt.Println(err.Error())
+	} else {
+		// Create Cache and check if CacheName exists
+		err := client.CreateCache(&momento.CreateCacheRequest{
+			CacheName: CacheName,
+		})
+		if err != nil {
```

Comment:
do you need to check if cache already exists in returned error and keep executing?

Reply:
I believe we want to throw an error if a cache already exists.

Reply:
https://github.com/momentohq/client-sdk-csharp/pull/30#discussion_r799816920

Reply:
right but do you want to not fail and keep going w/ example if cache already exists?

Reply:
for example like python example here.
https://github.com/momentohq/client-sdk-examples/blob/main/python/example.py#L41-L42

Reply:
Sorry, misunderstood your comment..
Updated here: https://github.com/momentohq/client-sdk-examples/blob/chore/addGoExample/golang/main.go#L35
Fail only other errors except `AlreadyExists` are returned.